sdk/cosmosdb/cosmos/src/client/SasToken/SasTokenAuthorizationHelper.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export class SasTokenAuthorizationHelper {
```

Comment:
remove this class and export the function directly:

`export function isSasToken ... `